probe/docker/tagger.go

```diff
@@ -10,9 +10,8 @@ import (
 
 // Node metadata keys.
 const (
-	ContainerID = "docker_container_id"
-	Domain      = "domain" // TODO this is ambiguous, be more specific
-	Name        = "name"   // TODO this is ambiguous, be more specific
+	ContainerID = report.DockerContainerID
+	Name        = report.Name // TODO this is ambiguous, be more specific
```

Comment:
I don't understand this comment

Reply:
I didn't write the original comment. I suspect it stems from the fact that key is "name" rather than, say, "container_name". So it's prone to clashing, and indeed does with `ImageName` and `ProcessName` that are also defined as "name". However, none of them are used in the same context. I am happy to get rid of the warning.